packages/network-enablement-controller/tsconfig.json

```
@@ -0,0 +1,22 @@
+{
+  "extends": "../../tsconfig.packages.json",
+  "compilerOptions": {
+    "baseUrl": "./",
+    "rootDir": "../.."
+  },
+  "references": [
+    {
+      "path": "../../packages/base-controller"
+    },
+    {
+      "path": "../../packages/network-controller"
+    },
+    {
+      "path": "../../packages/multichain-network-controller"
+    },
+    {
+      "path": "../../packages/controller-utils"
+    }
+  ],
+  "include": ["../../types", "./src", "tests/NetworkOrderController.test.ts"]
```

Nit: I believe you meant here
```suggestion
  "include": ["../../types", "./src", "tests/NetworkEnablementController.test.ts"]